adapters/bridge.go

```diff
@@ -77,13 +77,21 @@ func responseToRunResult(body []byte, input models.RunResult) models.RunResult {
 	return rr
 }
 
-func postToExternalAdapter(url models.WebURL, input models.RunResult) ([]byte, error) {
+func (ba *Bridge) postToExternalAdapter(input models.RunResult) ([]byte, error) {
 	in, err := json.Marshal(&bridgeOutgoing{input})
 	if err != nil {
 		return nil, fmt.Errorf("marshaling request body: %v", err)
 	}
 
-	resp, err := http.Post(url.String(), "application/json", bytes.NewBuffer(in))
+	request, err := http.NewRequest("POST", ba.URL.String(), bytes.NewBuffer(in))
+	if err != nil {
+		return nil, fmt.Errorf("building outgoing bridge http post: %v", err)
+	}
+	request.Header.Set("Authorization", "Bearer "+ba.BridgeType.OutgoingToken)
```

Comment:
This looks kind of awkward on the external adapter receiving the outgoing token. Would you have anything against removing the "Bearer " prefix?

Reply:
I’m doing it to comply with the HTTP 1.0 standard. I believe it to be standard practice. For example, this differentiates is from a basic auth header which looks like:

Auhtorization: Basic someinfo

https://security.stackexchange.com/questions/108662/why-is-bearer-required-before-the-token-in-authorization-header-in-a-http-re